Signal/src/contact/OWSContactsManager.m

```diff
@@ -7,6 +7,8 @@
 
 typedef BOOL (^ContactSearchBlock)(id, NSUInteger, BOOL *);
 
+NSString *const OWSContactsManagerContactsDidChangeNotification = @"OWSContactsManagerContactsDidChangeNotification";
```

Comment:
Ah, that makes sense. Thanks for clarifying - I've taken your suggestion.  

In the long run, I wonder if "signal contacts" is clearer than "signal recipients".